backend/src/packages/chaiNNer_external/external_stable_diffusion/automatic1111/upscaling.py

```diff
@@ -0,0 +1,201 @@
+from __future__ import annotations
+
+from enum import Enum
+
+import numpy as np
+from nodes.groups import Condition, if_enum_group, if_group
```

Comment:
isort won't like that. 

@mateusz Are you using VSCode? We set up isort to automatically run on save in VSCode, so I would like to know whether it didn't work for you or whether you're using another IDE.

```suggestion
import numpy as np

from nodes.groups import Condition, if_enum_group, if_group
```

Reply:
I'm using VSCode, but I must admit I haven't put serious thought into it and been installing random tools to get it to pass :) It might be related to Microsoft's "Black Formatter v2022.4.0" idiosyncracy?

In any case, I've then installed `isort v2022.8.0` from VSCode plugins too, and that did not reformat the file.
Then I installed `pip install isort` (isort-5.12.0) and ran the exact command your pipeline is running and the following fix was applied:

```diff
--- a/backend/src/packages/chaiNNer_external/external_stable_diffusion/automatic1111/upscaling.py
+++ b/backend/src/packages/chaiNNer_external/external_stable_diffusion/automatic1111/upscaling.py
@@ -3,6 +3,7 @@ from __future__ import annotations
 from enum import Enum
 
 import numpy as np
+
 from nodes.groups import Condition, if_enum_group, if_group
 from nodes.impl.external_stable_diffusion import (
     STABLE_DIFFUSION_EXTRA_SINGLE_IMAGE_PATH,
 ```

Reply:
That is weird.

> Black Formatter v2022.4.0
> isort v2022.8.0

Those are quite old versions. Maybe that's the issue?